tests/test_orgids_prefixes.py

```diff
@@ -0,0 +1,10 @@
+from libcove2.common import get_orgids_prefixes
+
+
+def test_get_orgids_prefixes():
```

Comment:
Might be worth a note that this test will fail if internet access or org id server  is down but we can do that later.